### ISSUE TITLE
Pagination event is raised twice on each click

### ISSUE THREAD

**Issue by jonest** [HealthCatalyst/Fabric.Cashmere#316]
**Describe the bug**
Event is raised twice when paging. See demo below.

**To Reproduce**
https://cashmere-demo-zkqvml.stackblitz.io
Open the console and you'll see each paging gets logged to the console twice.

**Expected behavior**
I'd expect it to only trigger the event once per click, to avoid making double the requests.

**Screenshots**

**Browser:**

*   Chrome - Version 67.0.3396.62 (Official Build) (64-bit)

**Comment by andrew-frueh** [comment]
Took a quick look this weekend and it appears that `set pageNumber()` gets called twice any time a button in the component is clicked, but only once when the value of pageNumber is set programmatically (via the textbox in the example)

**Comment by corykon** [comment]
Fix for this being merged into dev with the v3 branch
